am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/FilterResourceIntegrationTest.java

```diff
@@ -30,14 +32,27 @@ public void setupTest() {
         resourceId = UUID.randomUUID().toString();
     }
 
+    @Test
+    public void filterResource_whenAddTwoRecordsWithSameResourceIdAndUserIdButDifferentAttribute_ReturnsTwo() {
```

Comment:
Please remove `filterResource_` from method name. Feel free to apply same to other methods in touched test class. The reason is that that class will only contain tests targeting `filterResource` service method.

Reply:
👍